src/main/java/emu/grasscutter/documentation/DocumentationHandler.java

```diff
@@ -0,0 +1,9 @@
+package emu.grasscutter.documentation;
```

Comment:
The documentation package should be within ``emu.grasscutter.server.http``

Reply:
Done